examples/docs_snippets/docs_snippets/integrations/dbt/tutorial/upstream_assets/definitions.py

```diff
@@ -1,14 +1,14 @@
+# ruff: noqa: I001
```

Comment:
the different sections of this file correspond to different files in the dbt scaffold, right? I think this will be easier to maintain if we have a separate file for each of the files in the dbt scaffold that we want to include snippets of the tutorial.

this will:
- avoid putting code that's the user is supposed to put in a file named assets.py inside a file named definitions.py
- avoid accidentally missing a required import, because it's included further up